inbm/dispatcher-agent/dispatcher/fota/installer.py

```diff
@@ -78,7 +78,7 @@ def get_product_params(self, platform_product: str) -> Dict:
                 if ' ' in tool_name or tool_name.isspace():
                     raise FotaError(f"FOTA tool name cannot contain spaces - {tool_name}")
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Robustness$</h2>**
**Suggestion:** Validate the tool_name for null or empty strings before checking for spaces or calling isspace(). This prevents potential AttributeError when tool_name is None.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
                if tool_name and (' ' in tool_name or tool_name.isspace()):
                    raise FotaError(f"FOTA tool name cannot contain spaces - {tool_name}")
```